### ISSUE TITLE
Render Tiles in Windows - How?!?!?! 

### ISSUE THREAD

**Issue by Amrizhan** [mapnik/mapnik#3129]
Hello Guys,

since i Remeber, i want an own Tileserver.

I tried in Ubuntu, i Failed cause of my small Skills in Linux.

Now i am Trying it in Windows. So far so good, but i didnt reached my destination.

What i've done:

Postgre SQL DB
Postgis
osm2pgsql used
Result: There is Data in the Database.

But how can i render my Tiles?!

There are 2 Opinions:

Use Mapnik (Fine, there are 100 Manuals, but only to Create a Demo. I want not a demo, i want to render the data from my Database!)

Install Tilemill (So far so good, but what i do next? I Dont know where there is a use that this Program can connect to my Database)

Long Story Short Question:

How can i Render Tiles from my Database (Postgre SQL). Please with Real Examples and not only manuals where i can Create Demos.

Thank you very much!

**Comment by Amrizhan** [comment]
I downloaded world boundaries zipped data unzip all of them into my folder, Tried to run generate_xml.py and generate_tiles 
In generate_xml i have error: Can't find include templates
chage in source to /inc/*.template, but it not helps

in generate_tiles.py i had like required password cmd was next:
generate_tiles.py --dbname osm --host 'localhost' --user postgres --port 5432 --password

Pls help me!!!!

**Comment by Amrizhan** [comment]
Ok, last news 

Now i got error failed initialize projection

**Comment by Amrizhan** [comment]
I tried new mapnik scripts: 

And have also error:

C:\geodata\mapnik>generate_tiles.py -h
Traceback (most recent call last):
  File "C:\geodata\mapnik\generate_tiles.py", line 194, in <module>
    home = os.environ['HOME']
  File "C:\Python27\lib\os.py", line 425, in __getitem__
    return self.data[key.upper()]
KeyError: 'HOME'

what wrong on windows???

**Comment by Amrizhan** [comment]
Windows commands
set MAPNIK_MAP_FILE=/osm_tiles/ma_osm.xml
set MAPNIK_TILE_DIR=/osm_tiles
set HOME=/osm_tiles

need to put this commands on prompt

**Comment by springmeyer** [comment]
@Amrizhan - this site is for bugs in the mapnik software. It looks like you are having trouble setting up the osm stylesheets. I recommend you use the latest stylesheets over at https://github.com/gravitystorm/openstreetmap-carto/ and seek help from openstreetmap users.

However based on your last comment, perhaps you've figured out all the problems. If so, great work! Hope all your tile rendering goes well. If you hit any crashes then this repo is the right place, and feel free to create a new issue.

**Comment by Amrizhan** [comment]
@springmeyer - yes! Thank you!! I solve all problems, now i am steel doing render tiles, i choose 17 zoom level, and it takes too long, i Start on friday morning, and now thursday.
I made few mistakes. 1: i got a lot empty tiles 2. i creating tiles on one proc mode
